index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Glory Mugizi Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="tool-bar">
+          <li class="my-logo">MY LOGO</li>
+          <li class="toggle-menu">
+            <img src="images/humbger.png" alt="" srcset="" />
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Kindly wrap only the navbar in a `header` tag and not the whole headline section for better semantics 👍 
- [ ] Kindly remove the `ul` element and the `li` elements, they're not a list so it's better for semantics if we don't use the `ul` here.
- [ ] Kindly wrap the logo in an `a` tag with `href="#"`, this way the logo will take to the top of the page when we click on it 💯

Reply:
Hell @ha-manel,
The navbar was wrapped, ul element removed and logo wrapped in a with href="#".
Thank you.